wcivf/apps/api/tests/test_api.py

```diff
@@ -34,6 +35,8 @@ def test_candidates_for_postcode_view_raises_error(self):
 class TestAPISearchViews(APITestCase):
 
     def setUp(self):
+        cache.clear()
```

Comment:
This is fine, but I reckon the real solution is to use `django.core.cache.backends.dummy.DummyCache` in the settings the tests use. 

It might be a good idea to change the default cache to `DummyCache` and leave production to use Redis / whatever.